pkg/cloudprovider/suite_test.go

```diff
@@ -113,7 +115,41 @@ var _ = Describe("CloudProvider", func() {
 	var nodePool *corev1beta1.NodePool
 	var nodeClaim *corev1beta1.NodeClaim
 	var _ = BeforeEach(func() {
-		nodeClass = test.EC2NodeClass()
+		nodeClass = test.EC2NodeClass(
```

Comment:
optional: Did we consider automatically using these values when we are using the default EC2NodeClass from `test.EC2NodeClass`?

Reply:
I didn't want it to be surprising to developers that the status is already hydrated and they will need update the API and status.

Reply:
Discussed Offline, will add in a follow-up